README.md

```diff
@@ -837,6 +837,137 @@ OPTIONS:
+INFO[0000] Created new task execution role policy arn:aws:iam::aws_account_id:policy/amazon-ecs-cli-setup-myTaskExecutionRole-policy-20181023T210805Z
+INFO[0000] Attached AWS managed policy arn:aws:iam::aws:policy/service-role/AmazonECSTaskExecutionRolePolicy to role myTaskExecutionRole
+INFO[0001] Attached new policy arn:aws:iam::aws_account_id:policy/amazon-ecs-cli-setup-myTaskExecutionRole-policy-20181023T210805Z to role myTaskExecutionRole
+INFO[0001] Writing registry credential output to new file C:\Users\myuser\regcreds\regCredTest\ecs-registry-creds_20181023T210805Z.yml
+```
+
+The output file `ecs-registry-creds_20181023T210805Z.yml` should like like this:
+```
+version: "1"
+registry_credential_outputs:
+  task_execution_role: myTaskExecutionRole
+  container_credentials:
+    my-registry.example.com:
+      credentials_parameter: arn:aws:secretsmanager:region:aws_account_id:secret:amazon-ecs-cli-setup-my-registry.example.com-VeDqXm
+      container_names:
+      - web
+      - log
+```
+
+This file contains: 
+* the name of the IAM Task Execution Role with permissions for the new secrets 
+* the ARN of the new `credentials_parameter` created for the registry
+* the list of containers the new `credentials_parameter` should be used for when running a task or service
+
+We can now use this file with `ecs-cli compose up` to start a task with images in our private registry.
+
+### Using private registry credentials when launching tasks or services
+
+Now that we have an output file that identifies which resources we need to use our private registry, the ECS CLI will incorporate them into our Docker Compose project when we run `ecs-cli compose`.
+
+In the same directory (let's call it "privateImageApp"), create a docker-compose.yml file for your application:
+
+```
+version: "3"
+services:
+  web:
+    environment:
+      - SERVICE_NAME=web
+    image: my-registry.example.com/httpd 
+    ports:
+      - "80:80"
+  log:
+    environment:
+      - SERVICE_NAME=log
+    image: my-registry.example.com/logging
+    logging:
+      driver: awslogs
+      options:
+        awslogs-group: myApps
+        awslogs-region: us-west-2
+        awslogs-stream-prefix: privateImageApp
+```
+
+Now run the command `ecs-cli compose up` to launch a task. The ECS CLI will automatically detect and use the newest `ecs-registry-creds` file within the current directory:
+
+```
+$~\privateImageApp> ecs-cli compose up
+INFO[0000] Found ecs-registry-creds file C:\Users\myuser\regcreds\regCredTest\ecs-registry-creds_20181023T210805Z.yml
+INFO[0000] Using ecs-registry-creds value arn:aws:secretsmanager:region:aws_account_id:secret:amazon-ecs-cli-setup-my-registry.example.com-VeDqXm container name=web option name=credentials_parameter
+Using ecs-registry-creds value arn:aws:secretsmanager:region:aws_account_id:secret:amazon-ecs-cli-setup-my-registry.example.com-VeDqXm container name=log option name=credentials_parameter
```

Comment:
Sort of random, I should have noticed this in a previous PR, but I think its weird how it says "Using ecs-registry-creds value" and "Found ecs-registry-creds file". Why not replace "ecs-registry-creds" with "registry credentials". To me, "ecs-registry-creds" with the dashes makes it seem like that's a field or some other sort of technical thing which I should recognize, but its not right? An "ecs-registry-creds" isn't a thing right?

Reply:
I was following the [convention we use](https://github.com/aws/amazon-ecs-cli/blob/master/ecs-cli/modules/utils/compose/convert_task_definition.go#L281) to refer to values from `ecs-params.yml`. When we override docker-compose values in a task def with values from this file, we message the customer in a similar way: "Using ecs-params value as override" / "Using ecs-params value as override (was %v but is now %v)"

I'm not married to "ecs-registry-creds" as a term, but it is the (sans timestamp) name of the src file and I think it's helpful to indicate where the value came from. 

However, if we want to avoid creating a new convention for now I think omitting the term would be OK too b/c when those values are used we do identify the container & task def param (credentials_parameter) being updated.

Reply:
Ah I see, I guess I'm fine with leaving it the way it is